okhttp/src/main/java/okhttp3/CertificatePinner.kt

```diff
@@ -276,6 +268,31 @@ class CertificatePinner internal constructor(
     }
 
     override fun toString(): String = hashAlgorithm + hash.base64()
+
+    companion object {
+      @JvmStatic
+      fun newPin(pattern: String, pin: String): Pin {
```

Comment:
make this into the class constructor? If it’s not a data class it’s easy to write that way